src/components/judges-section.tsx

```diff
@@ -0,0 +1,33 @@
+import { judgesList } from "@/constants/judgesList";
+import Image from "next/image";
+import SectionHeading from "./section-heading";
+
+export const JudgesSection = () => {
+  return (
+    <section className="px-6 pt-18">
+      <div className="mx-auto flex max-w-4xl flex-col gap-12 px-6 md:px-12">
+        <SectionHeading>プロポーザル選考委員</SectionHeading>
+        <ul className="grid grid-cols-[repeat(auto-fit,192px)] justify-center gap-x-14 gap-y-8 px-10 md:grid-cols-[repeat(auto-fit,224px)]">
+          {judgesList.map(({ name, image, href }) => (
+            <li key={name} className="relative aspect-square h-full w-full">
```

Comment:
li の横幅は親の grid-template-columns で指定されているので w-full は不要です。また、親要素は高さを持たないので h-full も不要です。
aspect-square も li に対してつけるものではないので消してしまってOKです！

```suggestion
            <li key={name}>
```